app/src/main/res/values/strings.xml

```diff
@@ -1920,7 +1920,7 @@
     <string name="pocket_stories_placeholder_text">Discover more</string>
     <!-- Title of an app feature. Smaller than a heading.-->
     <string name="pocket_stories_feature_title">Powered by Pocket</string>
-    <!-- Caption which concatenates the text of "pocket_stories_feature_learn_more"  -->
+    <!-- Caption for describing a certain feature. The placeholder is for a clickable text (eg: Learn more) which will load an url in a new tab when clicked.  -->
```

Comment:
I think we could make this better. "a certain feature" is a bit vague here.

```suggestion
    <!-- Placeholder caption for clickable text (eg: Learn more, which will load an url in a new tab when clicked) in the Pocket feature section of the home page. -->
```

Reply:
This string is not a placeholder for clickable text. It is a text containing a placeholder which will be clickable.

If we are to combine the two and tie this string to the app feature we would get something like
"Caption for the Pocket recommended stories section of the home screen. Includes a placeholder for clickable text (eg: Learn more) which will load an url in a new tab when clicked"

But I wanted the description to be a bit vague as I don't expect localizers to necessarily be users of the app and know about it's features, where they are used - on the home screen and not sure why this would matter.
Curious if we should offer this much info or if we should avoid it.

@Delphine which of the 3 descriptions (already in PR / from Elise / from this comment) should be used?

Reply:
That totally makes sense. Let's stick with what you have 👍

Reply:
For the record. Localizers are users of the app and are often core fans. They are not payed to do this, they are volunteers. And they need to understand in context what they are translating.

Reply:
Is there no onboarding for localization and working with communtiy?

Reply:
Thanks for paying attention to string comments, they are very helpful for localizers!

If you look at [this example](https://pontoon.mozilla.org/sl/firefox/all-resources/?string=190587) in Firefox, a comment like "%s will be replaced with pocket_stories_feature_learn_more" would already be enough.

If you also want to explain where the string is used, I'd join it with your proposal in the comment:
"Caption for the Pocket recommended stories section of the home screen. %s will be replaced with pocket_stories_feature_learn_more."

I'd refrain from quoting the string explicitly in the comment (even as an example), because that gets outdated easily.